Comments/ClientApp/src/components/GenerateCode/GenerateCode.scss

```diff
@@ -2,6 +2,7 @@
   &__row {
     display: flex;
     align-items: center;
+    padding: 10px;
```

Comment:
the pixels here is a bit concerning. does that look right on every screensize?

how about using a sass variable from here: https://nice-digital.github.io/nice-design-system/foundations/spacing/#spacing-scale